manifests/init.pp

```diff
@@ -136,7 +136,7 @@
   $redis_slave_output_buffer_soft_limit = $redis::params::redis_slave_output_buffer_soft_limit,
   $redis_slave_output_buffer_soft_limit_max_interval = $redis::params::redis_slave_output_buffer_soft_limit_max_interval,
   $redis_snapshotting = $redis::params::redis_snapshotting,
-  $manage_config_file = $redis::params::manage_config_file
```

Comment:
Upstream does not have the manage_config_file option. I'm not sure that our changes are more likely to be accepted with two new variables than with one.